src/script/entity/message/Message.ts

```diff
@@ -70,6 +77,7 @@ export class Message {
   public readReceipts: ko.ObservableArray<ReadReceipt>;
   public super_type: SuperType;
   public type: string;
+  reaction: any;
```

Comment:
can't we have a type for this?

Reply:
Yeah, this is probably `ReactionType` (from `@wireapp/core/dist/conversation`).